2018-01/jan-0.md

```diff
@@ -0,0 +1,52 @@
+## Babel Weekly Notes - 2018-01-01
+
+## Participant
+
+- Logan Smyth
+- Nicolo Ribaudo
+- Sven Sauleau
+
+## Turn on Slack inviter again
+
+## Babelrc lookup
+
+https://github.com/babel/babel/issues/6766
+
+- Logan: not using the babelrc in the node_modules by default, opt-in
+- Opt-in with the overide key ?
+- Sven: idea about using babel-preset-env in the root configuration and transpile deps against the target
+
+## Plugin order
+
+- Still priority? probably not a blocker for the 7 release
+
+## Async plugins/visitors
+
+- Not for now
+- Would explose if Babel supports async for plugins or visitor
```

Comment:
`expose`.

Reply:
Maybe also add a line about babel-register being a big blocker for this?